### ISSUE TITLE
[Feature Request]: Hand Gesture

### ISSUE THREAD

**Issue by Soumyanegi1** [hasmithagunda123/ALL_INDIA_HACKATHON#91]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

If we are talking about the safety , then we should definite add the hand gesture movement. 
For Example:- There are 3-4 peoples trying to cover the girl/boy face then there that person can use the hand movement so that it can gave the signal.

### Use Case

We can use this feature to add on more efficient to this project and it will work more scalable.

### Benefits

Implementing this feature will make this project more scalable and efficient.

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [x] I want to work on this issue

**Comment by Wahid7852** [comment]
a special gesture would give it away and would alert the attackers, a normal gesture would be captured too many times
need to look into it more

**Comment by Soumyanegi1** [comment]
it can be easy hand gesture so that attackers don't found it suspicious.

**Comment by Wahid7852** [comment]
Yes but for it to be easy, unsuspecting and uncommon is difficult

We need to discuss properly cause one unintended signal and false alarm
rates will spike

I've an idea which we can talk about on discord if you want

**Comment by Soumyanegi1** [comment]
yes sure!
